test/unit/core_test.rb

```diff
@@ -8,25 +8,6 @@ class Superstore::CoreTest < Superstore::TestCase
     assert !issue.destroyed?
   end
 
-  test 'initialize yields self' do
-    issue = Issue.new { |i| i.description = 'bar' }
-    assert_equal 'bar', issue.description
-  end
-
-  test 'dup' do
```

Comment:
Do these tests still pass?  Why remove them?

Reply:
`initialize yields self` would still pass. The initialize code is no longer implemented in superstore. I don't want to test active record.